evalset/multiopt_test_funcs.py

```diff
@@ -0,0 +1,25 @@
+from evalset.test_funcs import TestFunction, lzip
+import numpy
+
+class MultioptTestFunction(TestFunction):
+
+  def __init__(self, dim):
+    super(MultioptTestFunction, self).__init__(dim)
+    self.local_minima = [] # Sorted in increasing order of function value at the local minima
```

Comment:
I should rename this as `local_minima_loc` (analogous to `min_loc`) in the original `evalset`.